src/gui/AboutDialog.cpp

```diff
@@ -52,9 +52,12 @@ AboutDialog::AboutDialog(QWidget* parent) :
 	copyrightLabel->setText( copyrightLabel->text().
 					arg( LMMS_PROJECT_COPYRIGHT ) );
 
-	authorLabel->setPlainText( embed::getText( "AUTHORS" ) );
+	authorLabel->setPlainText(
+					embed::getText( "AUTHORS" ) );
 
-	licenseLabel->setPlainText( embed::getText( "LICENSE.txt" ) );
+	/*involvedLabel->setPlainText(
```

Comment:
Why did you disable the contributors file, it contains all of our github contributors and it's more accurate than the authors file (though the authors file contains some people that wouldn't be credited through github)

Reply:
> it contains all of our github contributors

@Umcaruje Not anymore, jasp00 removed the CMake logic that created the file in 30f1e52be35ed35f8e0e40f876620ce49007be62. He planned to replace that by a maintenance task on our server, but it appears it was never implemented correctly. That's probably why @Sawuare removed it. Related: LMMS/lmms.io/pull/198, #3016

Reply:
Then, should we update the authors file with some missing people? Or revert @jasp00's changes, since he sent a mail saying he won't be able to contribute for some time.

Reply:
I didn't know the story of the contributors tab. I just removed it because it didn't display anything other than: "Contributors ordered by number of commit:".